src/main/java/org/apache/sling/feature/cpconverter/acl/DefaultAclManager.java

```diff
@@ -134,7 +134,7 @@ public void addRepoinitExtension(List<VaultPackageAssembler> packageAssemblers,
             String text = formatter.toString();
 
             if (!text.isEmpty()) {
-                featureManager.addOrAppendRepoInitExtension(text);
+                featureManager.addOrAppendRepoInitExtension(text, null);
```

Comment:
Would it make sense to add an extra override to addOrAppendRepoInitExtension() with a single argument that calls the other one with `null` as the second argument? That way we don't have to specify `null` everywhere in the code where we don't need runmodes?

Reply:
I was thinking about that but tbh it makes it easier to miss that you might need to pass a runmode (what made me miss the need to pass it earlier). if you still insist I certainly can add a corresopnding alternative wrapping signature and replace the null calls.